go/vt/vtctl/vtctl.go

```diff
@@ -378,6 +378,9 @@ var commands = []commandGroup{
 			{"ListTablets", commandListTablets,
 				"<tablet alias> ...",
 				"Lists specified tablets in an awk-friendly way."},
+			{"ListShardRanges", commandListShardRanges,
```

Comment:
nit - Maybe "GenerateShardRanges" ? I think List reads as something that already exists.